pybamm/models/full_battery_models/lead_acid/basic_full.py

```diff
@@ -0,0 +1,286 @@
+#
+# Basic lead-acid model
+#
+import pybamm
+from .base_lead_acid_model import BaseModel
+
+
+class BasicFull(BaseModel):
+    """Porous electrode model for lead-acid, from [2]_.
+
+    This class differs from the :class:`pybamm.lead_acid.Full` model class in that it
+    shows the whole model in a single class. This comes at the cost of flexibility in
+    comparing different physical effects, and in general the main DFN class should be
+    used instead.
+
+    Parameters
+    ----------
+    name : str, optional
+        The name of the model.
+
+    References
+    ----------
+    .. [2] V Sulzer, SJ Chapman, CP Please, DA Howey, and CW Monroe. Faster lead-acid
```

Comment:
Minor typo: should the reference be [1]?

Reply:
It has to be a different number because "full" and "basicfull" are in the same doc file and sphinx doesn't like it if the same ref number is used twice